doc/en/user/source/configuration/virtual-services.rst

```diff
@@ -125,3 +127,62 @@ It is possible to completely restrict access to the global OWS services by setti
 To disable global services log into the GeoServer web administration interface and navigate to "Global Settings". Uncheck the "Enable Global Services" check box.
 
    .. figure:: img/global-services.jpg
+
+.. _workspace_isolated:
+
+Isolated Workspaces
```

Comment:
Not sure why this ended up in a PR about the welcome message.

Reply:
I did not write this, it was removed from the workspace reference page.